org.mavlink.qgroundcontrol.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.mavlink.qgroundcontrol
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: QGroundControl
+
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --socket=pulseaudio
+- --device=dri
+- --share=ipc
+- --share=network
+
+modules:
+  - name: QGroundControl
+    sources:
+    - type: git
+      url: https://github.com/mavlink/qgroundcontrol.git
+      commit: eca2614b6e2a237c3717f246c8c6e42d7d903aaf # Mar 1, 2023
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+    - -D BUILD_SHARED_LIBS=OFF
+    - -D OpenGL_GL_PREFERENCE=GLVND
+    cleanup:
+    - /include
+    - /bin/dbf*
+    - /bin/shp*
+    - /lib/cmake
+    - /share/shapelib
+    - "*.a"
+    - "*.cmake"
+    - /share/metainfo/org.mavlink.qgroundcontrol.metainfo.xml
```

Comment:
why is this in the cleanup rules?

Reply:
Because this file is not used.

Reply:
if there is one upstream, there is no reason to have one in this repository. Use a patch if needed.

Reply:
in any case, cleanup is NOT the place for this specific use.

Reply:
I removed that line.